implants/golem/src/main.rs

```diff
@@ -1,3 +1,5 @@
+#![deny(warnings)]
```

Comment:
do we need this? it seems to defeat the purpose of having lint blocking PR.

Reply:
Yes it causes cargo to fail any builds that compile with warnings